app/com/linkedin/drelephant/tuning/engine/SparkHBTParamRecommender.java

```diff
@@ -0,0 +1,361 @@
+package com.linkedin.drelephant.tuning.engine;
```

Comment:
There is some logic in https://github.com/edwinalu/dr-elephant/pull/2 for suggesting recommended values. What would be the best way to merge some of the logic in that PR, with this PR? Should the logic be kept in the spark directory (with SparkExecutionEngine getting the values from there), or moved to tuning? How would text recommendations for application modifications be shown to users?

Reply:
It suggestions are part of some Heuristic then it can be referred in this class and return from this class without any modification.